ci/deps/actions-38-minimum_versions.yaml

```diff
@@ -54,3 +55,6 @@ dependencies:
   - xlrd=2.0.1
   - xlsxwriter=1.4.3
   - zstandard=0.15.2
+
+  - pip:
+    - PyQt5>=5.15.1
```

Comment:
Why is this `>=`? This build we want to test the minimum versions exactly

Reply:
Typo will change it.